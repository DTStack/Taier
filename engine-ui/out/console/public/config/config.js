/**
 * 应用全局配置文件
 */
window.APP_CONF = {
    prefix: '', // 应用前缀
    name: 'Console', // 应用名字
    loadingTitle: 'Console · 控制台', // loading的应用名字
    logo: 'public/img/logo.svg', // header图标
    titleName: 'Console · 控制台', // 网页titletheme: 'default', // 主题
    hideUserCenter: false, // 隐藏用户中心按钮
    disableHelp: false, // 禁用帮助文档
    UIC_URL: 'http://dtuic.dtstack.cn', // UIC中心地址
    UIC_DOMAIN: '.dtstack.cn', // UIC域名
    closeBrowserCheck: false, // 是否关闭浏览器检查，默认为 false
    macChrome: 'https://dl.google.com/chrome/mac/stable/GGRO/googlechrome.dmg', // mac版本Chrome下载地址
    windowsChrome: 'https://dl.google.com/tag/s/appguid%3D%7B8A69D345-D564-463C-AFF1-A69D9E530F96%7D%26iid%3D%7B8A1ECDAD-CD48-3F61-7773-5D75AC6B409A%7D%26lang%3Dzh-CN%26browser%3D4%26usagestats%3D1%26appname%3DGoogle%2520Chrome%26needsadmin%3Dprefers%26ap%3Dx64-stable-statsdef_1%26brand%3DCHBD%26installdataindex%3Dempty/update2/installers/ChromeSetup.exe' // windows版本chrome下载地址
}
