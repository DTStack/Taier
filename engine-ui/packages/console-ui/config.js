'use strict';
module.exports = {
    server: {
        port: 8080,
        host: 'rdos.yarn.dtstack.net',
        proxy: {
            "/api": {
                target: "http://172.16.8.104:9020",
                // target: "http://172.16.8.101:9020",
                // target: "http://172.16.6.207:9020", // tmp server
                // ignorePath: true, 
                changeOrigin: true,
                secure: false,
            },
            "/uic": { // UIC地址
                target: "http://dtuic.dtstack.net",
                changeOrigin: true,
                secure: false,
            }
        }
    }
};