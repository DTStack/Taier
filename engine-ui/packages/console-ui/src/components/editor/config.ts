export const defaultOptions: any = {
    language: 'sql',
    readOnly: false,
    contextmenu: true,
    autoIndent: true,
    automaticLayout: true,
    showFoldingControls: 'always',
    folding: true,
    foldingStrategy: 'auto',
    suggestFontSize: 13,
    fontSize: 13,
    iconsInSuggestions: false,
    fixedOverflowWidgets: true,
    renderControlCharacters: true
}
