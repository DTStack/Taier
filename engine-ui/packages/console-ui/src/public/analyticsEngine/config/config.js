/**
 * 应用全局配置文件
 */
window.APP_CONF = {
    name: 'AnalyticEngine', // 应用名字
    titleName: 'AnalyticEngine · 分析引擎', // 网页title
    logo: 'public/analyticsEngine/img/logo.svg', // header图标
    loadingTitle: 'AnalyticEngine · 分析引擎' // loading的应用名字
}
