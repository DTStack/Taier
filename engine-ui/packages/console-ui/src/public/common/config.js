/**
 * 应用公用配置
 */
window.COMMON_CONF = {
    UIC_URL: 'http://uic.dtstack.net', // UIC中心地址
    UIC_DOMAIN: '.dtstack.net', // UIC域名
    prefix: 'DTinsight', // 应用前缀
    name: '数栈', // 应用名字
    titleName: '数栈', // 网页title
    loadingTitle: '数栈', // loading的应用名字
    theme: 'default', // 主题
    hideUserCenter: false, // 隐藏用户中心按钮
    disableHelp: false, // 禁用帮助文档
    macChrome: 'https://dl.google.com/chrome/mac/stable/GGRO/googlechrome.dmg', // mac版本Chrome下载地址
    windowsChrome: 'https://dl.google.com/tag/s/appguid%3D%7B8A69D345-D564-463C-AFF1-A69D9E530F96%7D%26iid%3D%7B8A1ECDAD-CD48-3F61-7773-5D75AC6B409A%7D%26lang%3Dzh-CN%26browser%3D4%26usagestats%3D1%26appname%3DGoogle%2520Chrome%26needsadmin%3Dprefers%26ap%3Dx64-stable-statsdef_1%26brand%3DCHBD%26installdataindex%3Dempty/update2/installers/ChromeSetup.exe' // windows版本chrome下载地址
}
