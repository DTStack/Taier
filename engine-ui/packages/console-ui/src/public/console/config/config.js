/**
 * 应用全局配置文件
 */
window.APP_CONF = {
    name: 'Console', // 应用名字
    loadingTitle: '控制台', // loading的应用名字
    logo: 'public/console/img/logo.svg', // header图标
    titleName: '控制台' // 网页title
}
