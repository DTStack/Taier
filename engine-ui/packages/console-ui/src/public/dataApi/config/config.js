/**
 * 应用全局配置文件
 */
window.APP_CONF = {
    name: 'DataAPI', // 应用名字
    loadingTitle: 'DataAPI · 数据API', // loading的应用名字
    logo: 'public/dataApi/img/logo.svg', // header图标
    titleName: 'DataAPI · 数据API' // 网页title
}
