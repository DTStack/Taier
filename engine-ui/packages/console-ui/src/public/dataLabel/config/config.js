/**
 * 应用全局配置文件
 */
window.APP_CONF = {
    name: 'Label', // 应用名字
    loadingTitle: '标签工厂', // loading的应用名字
    titleName: '标签工厂' // 网页title
}
