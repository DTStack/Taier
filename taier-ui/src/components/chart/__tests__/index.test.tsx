import { render } from '@testing-library/react';
import config from './fixtures/config';

describe('Test Chart Component', () => {
    beforeAll(() => {
        jest.useFakeTimers('modern').setSystemTime(new Date('2020-01-01'));
    });

    afterAll(() => {
        jest.useRealTimers();
    });

    it('Should match snapshot', () => {
        import('../index').then((Chart) => {
            const { asFragment } = render(<Chart.default option={config} width={110} height="110px" />);

            expect(asFragment()).toMatchSnapshot();
        });
    });
});
