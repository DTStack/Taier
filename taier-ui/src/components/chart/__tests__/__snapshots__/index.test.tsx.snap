// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Chart Component Should match snapshot 1`] = `
<DocumentFragment>
  <div
    _echarts_instance_="ec_1577836800000"
    style="width: 110px; height: 110px; user-select: none; background: transparent;"
  >
    <div
      style="position: relative; overflow: hidden; width: 110px; height: 110px; padding: 0px; margin: 0px; border-width: 0px;"
    >
      <canvas
        data-zr-dom-id="zr_0"
        height="110"
        style="position: absolute; left: 0px; top: 0px; width: 110px; height: 110px; user-select: none; padding: 0px; margin: 0px; border-width: 0px;"
        width="110"
      />
    </div>
  </div>
</DocumentFragment>
`;
